### PR TITLE
Clean glTF examples

### PR DESCRIPTION
**Pull request by Troy-Ferrell**
## Overview

This change removes the boombox & lantern gltf files which account for almost a majority of the file size of the examples unitypackage. The boombox is not used at all as the glb example scene downloads the gltf via the web url. The lantern gltf has been replaced with a simpler duck one. 

The gltf loading example scene also copies all the models into the streaming assets project folder with an onvalidate() call. The user has no knowledge of this. The editor inspector for this example scene now performs this actions with a button and debug information has been added to alert users that they need to put gltf files in streaming assets. 

## Changes
- Fixes: #5256 

## Verification
Thanks to @Cameron-Micka for his dependency window work. It helped me find these unused assets.

### CODE REVIEW

File: Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Duck/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Duck
+## Screenshot
+
+![screenshot](screenshot.png)
+
+## License Information
+
+Copyright 2006 Sony Computer Entertainment Inc.
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Duck/README.md]: before merging this, please get approval internally and register the usage (i can help)

Comment (Cameron-Micka) [reply Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Duck/README.md]: The duck makes it's triumphant return! https://www.youtube.com/watch?v=7UaDvIPpQnQ

Comment (Troy-Ferrell) [reply Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Duck/README.md]: "Retro" is in these days :P

Comment (Troy-Ferrell) [reply Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Duck/README.md]: @davidkline-ms I switched out the duck model for a microsoft avocado model. 

@Cameron-Micka sorry, the duck won't be making a triumphant return :(

**Comment by wiwei** [comment]
@Troy-Ferrel can you take a look at the Standalone build failures?


Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs(14,54): error CS0234: The type or namespace name 'Editor' does not exist in the namespace 'UnityEditor' (are you missing an assembly reference?)
 
(Filename: Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs Line: 14)

Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs(13,6): error CS0246: The type or namespace name 'CustomEditorAttribute' could not be found (are you missing a using directive or an assembly reference?)
 
(Filename: Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs Line: 13)

Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs(13,6): error CS0246: The type or namespace name 'CustomEditor' could not be found (are you missing a using directive or an assembly reference?)
 
(Filename: Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs Line: 13)

Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs(16,30): error CS0115: 'TestGltfLoadingEditor.OnInspectorGUI()': no suitable method found to override
 
(Filename: Assets\MixedRealityToolkit.Examples\Demos\Gltf\Scripts\Editor\TestGltfLoadingEditor.cs Line: 16)

**Comment by Troy-Ferrell** [comment]
@wiwei  Yes, I saw that. I am confused because I thought anything in "Editor" folders shouldn't build except for the editor

**Pull request merged**